ui/hooks/useTransactionInsights.js

```diff
@@ -98,7 +110,7 @@ const useTransactionInsights = ({ txData }) => {
     insightComponent = (
       <Tab
         className="confirm-page-container-content__tab"
-        name={getSnapName(selectedSnap?.id, subjectMetadata[selectedSnap?.id])}
+        name={getSnapName(selectedSnap?.id)}
```

Comment:
Can this be undefined ? Before this would never be undefined 🤔

Reply:
Yeah, technically it should never be undefined because at that point there would be some insight snap in existence.